elastic/elastic.go

```diff
@@ -21,13 +20,8 @@ func GetConnection(host string, port int) *elastic.Client {
 			Transport: elasticsearch.WrapRoundTripper(http.DefaultTransport),
 		}),
 	)
-	if err != nil {
-		logrus.Error("connect to elastic failed")
-		panic(err)
-	}
 
-	logrus.Infof("successfully connected to elastic : %s", connection)
```

Comment:
why did you deleted infos

Reply:
these helper functions only return errors to the caller, caller decides if there are no errors whether to log a success message or not.